Doc/cmus.txt

```diff
@@ -516,6 +516,12 @@ pl-import [filename]
 pl-rename <name>
 	Renames the selected playlist.
 
+pl-delete [-a] <name>
+	Deletes the playlist with the given name.
+
+	-a
+		Deletes all playlists
```

Comment:
\+ just noticed pl-delete isn't keeping to alphabetical order

Reply:
3a4ca11